canonical-web-and-design/ubuntu.com
# Add tutorials to the nav + update urls

## Done

- Add tutorials to the footer
- Add tutorials secondary nav
- Update all old tutorials.u.c urls to /tutorials

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- On small screens, see that tutorials is in the header
- See that all the references (e.g. Download mega menu) now points to the /tutorials urls


## Issue / Card

Fixes #6538
